mne/channels/channels.py

```diff
@@ -950,6 +950,80 @@ def interpolate_bads(
 
         return self
 
+    def interpolate_to(self, montage, method="MNE", reg=0.0):
+        """Interpolate data onto a new montage.
```

Comment:
say for now that it only supports EEG. But technically it could be used for other modalities like fNIRS, ECoG etc. See the test_interpolation.py file to see what modality is supported. To address here or we'll be able to add parameter ch_type once we support more than EEG.